### PR TITLE
Issue templates

### PR DESCRIPTION
**Pull request by kunicmarko20**
As explained here https://blog.github.com/2018-05-02-issue-template-improvements/

### CODE REVIEW

**Comment by kunicmarko20** [comment]
Just implemented it in my bundle: https://github.com/kunicmarko20/SonataAnnotationBundle/issues/new/choose

screenshot from 2018-05-08 10-12-53

**Comment by kunicmarko20** [comment]
cc @Soullivaneuh

File: project/.github/ISSUE_TEMPLATE/Question.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: ❓ Support Question
```
Comment (covex-nn) [inline project/.github/ISSUE_TEMPLATE/Question.md]: Usually users get the following answer to the support question:

> Hi, we try to keep Github issues for bug reports and feature requests only. If you have a question, please ask it on Stack Overflow or on #sonata on [the symfony-devs slack](https://symfony.com/slack-invite).

See https://github.com/sonata-project/SonataPageBundle/issues/985 for example. May be `Question.md` should be removed from PR?

Comment (kunicmarko20) [reply project/.github/ISSUE_TEMPLATE/Question.md]: Check the content of the template 😄

Comment (kunicmarko20) [reply project/.github/ISSUE_TEMPLATE/Question.md]: Hm, maybe, I don’t know really, what do rest of @sonata-project/contributors think?

Comment (covex-nn) [reply project/.github/ISSUE_TEMPLATE/Question.md]: I read only one line first, yes! )) and wanted to delete comment ) But i imagined, that i have a question; I click `New issue`; I see a beautiful button `Getting started` near `Support question` header with ❓ and  💬 emojis. I will not read weird message about "we try to keep", i will delete it and ask my question on github, then i will get a comment from @sonata-project and my issue will be closed =)

Comment (greg0ire) [reply project/.github/ISSUE_TEMPLATE/Question.md]: Maybe use ⛔ in the title then?

Comment (kunicmarko20) [reply project/.github/ISSUE_TEMPLATE/Question.md]: ⛔ NO support questions
use slack or stackoverflow.

?

**Comment by kunicmarko20** [comment]
Is it ok now?

**Comment by OskarStark** [comment]
Do we still need the `.github/ISSUE_TEMPLATE.md` ?

**Comment by kunicmarko20** [comment]
Yes, as default template if they decide not to choose anything

**Pull request merged**